river/metrics/test_metrics.py

```diff
@@ -13,6 +13,8 @@
 
 from river import metrics, utils
 
+from .test_wrappers import roc_auc_score as wrapper_roc_auc_score
```

Comment:
I would place the `wrapper_roc_auc_score` function in the `test_metrics.py` file, since it's only used there. Fine with you?

Reply:
Sure, no problem ;)